web/src/styles/app/table.css

```diff
@@ -120,11 +120,14 @@
   visibility: hidden;
   width: 0;
   border-right: 0px; /* ie fix */
+  padding-left: 0;
+  padding-right: 0;
 }
 
 .table-frozen-data tr > th:first-child,
 .table-frozen-data tr > td:first-child {
-  min-width: 140px; /* same as table-frozen-col-pane; for some reason just width doesn't work */
+  width: 140px; /* same as table-frozen-col-pane; only works on fixed tables */
```

Comment:
Interesting!  So td `width` is ignored (or somehow not honored) on normal tables?

Reply:
Yeah, I couldn't find a great explanation for it. My hunch is that because tables normally flex with the width of their cells' contents, the only way to set a firm width is to use the `display: fixed` setting for tables. Other tables seem to regard it as just a suggestion.